WindowsDevicePortalWrapper/UnitTestProject/WDPMockImplementations/WebSocket.cs

```diff
@@ -37,6 +37,11 @@ internal partial class WebSocket<T>
         private Func<object, X509Certificate, X509Chain, SslPolicyErrors, bool> serverCertificateValidationHandler;
 
         /// <summary>
+        /// The connection to the websocket.
+        /// </summary>
+        private Task<HttpResponseMessage> webSocketTask;
```

Comment:
You're adding an IDisposable member - shouldn't you add an IDisposable implementation to this class? That being said, there was already an IDisposable member, so perhaps it's unnecessary?